1.0/learn/how-to-document-ballerina-code/index.html

```diff
@@ -274,7 +274,7 @@ <h2 id="writing-ballerina-documentation">Writing Ballerina Documentation</h2>
 </code></pre></div></div>
 
 <blockquote>
-    <p><strong>Tip:</strong> Always, add a full stop at the end of a function description. However, as a best practice, omit the full stop if you have only one sentence in the parameter and return type descriptions. If there are multiple sentences for a parameter or return type description, add the full stop at the end of the first sentence and in the succeding ones. For example,</p>
+    <p><strong>Tip:</strong> Always, add a full stop at the end of all function descriptions. However, for the parameter and return type descriptions, omit the full stop if you have only one sentence. If there are multiple sentences, add the full stop at the end of the first sentence and in the succeeding ones. For example,</p>
```

Comment:
I think we can keep "a function description". We are talking about a single function here, right?

Reply:
How about
```suggestion
    <p><strong>Tip:</strong> Always, add a full stop at the end of a function description. However, for the parameter and return type descriptions, omit the full stop if you have only one sentence. If there are multiple sentences, add the full stop at the end of each sentence. For example,</p>
```